pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -446,3 +460,12 @@ func validateValues(values []string) error {
 	}
 	return nil
 }
+
+func isValidResourcePropertyName(resourcePropertyName string) bool {
```

Comment:
Hi Arvind! There's actually one more part in the property name, e.g., `resources.kubernetes-fleet.io/total-cpu`.